tests/Oryx.BuildImage.Tests/Node/NodeAppOutputDirTest.cs

```diff
@@ -27,7 +27,7 @@ public NodeAppOutputDirTest(ITestOutputHelper output) : base(output)
         // [InlineData("angular14", "dist")]
         // Temporarily blocking next app as next build is failing accross npm
         // [InlineData("blog-starter-nextjs", ".next")]
-        [InlineData("hackernews-nuxtjs", ".nuxt")]
+        // [InlineData("hackernews-nuxtjs", ".nuxt")]
```

Comment:
Does the `hackernews-nuxtjs` application not work with node 18?

Reply:
This app was failing and not working when tested it last time, and it's not for node 18 but fail on other node versions.